doc/apps/x509v3_config.pod

```diff
@@ -353,6 +353,15 @@ The B<ia5org> option changes the type of the I<organization> field. In RFC2459
 it can only be of type DisplayText. In RFC3280 IA5Strring is also permissible.
 Some software (for example some versions of MSIE) may require ia5org.
 
+ASN1 type of explicitText can be specified by prepending B<UTF8>,
+B<BMP> or B<VISIBLE> prefix followed by colon.
+
+Example:
+
+ [notice]
+
+ explicitText="UTF8:Explicit Text Here"
```

Comment:
Errr, no, don't indent.  That's never done in the config files, why should it here?  The one space indent is enough to make this a codeblock.